README.md

```diff
@@ -1,6 +1,132 @@
 # fullstory-segment-middleware-android
 
 ## Note: This project is currently under development. 
-FullStory Segment Middleware is a project demonstrating the full potential of joining forces between Segment + FullStory.
+--- 
+Segment is a customer data platform that unifies data collection and provides data to every team in your company. The middleware is a easy way to integrate FullStory with the Segment Analytics for Android SDK.
 
-Enables sending Segment Analytics data to FullStory and adding FullStory session replay links to Segment events.
\ No newline at end of file
+With minimal code change, the FullStory Segment Middleware provide developers the ability to send Segment Analytics data to FullStory, and adding FullStory session replay links to Segment events.
+
+<br/>
```

Comment:
Please remove the `<br>` tags - they introduce visual artifacts:

![image](https://user-images.githubusercontent.com/45576380/97488366-8842b900-1934-11eb-81d8-aea8d6e926de.png)

Markdown will maintain whitespace, so they aren't needed if that's your goal

Reply:
Thanks for the detailed review! VSCode had discrepancies in how md is previewed vs displayed in github, addressed them and updated my VSCode md preview with linting - should help with formatting markdown files.